NuGet.props

```diff
@@ -0,0 +1,24 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+    <Description>Advanced testing library for F#</Description>
+    <Authors>Anthony Lloyd, Henrik Feldt and contributors</Authors>
+    <Owners>Anthony Lloyd &amp; Henrik Feldt (cloned from Fuchu by @mausch)</Owners>
+    <Copyright>Copyright 2019</Copyright>
+    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
```

Comment:
Is there any reason for this one?

Expecto isn't using any restrictive or unusual license to merit requiring it to be approved.

Reply:
Not really any good reason, no.